packages/core/content-type-builder/admin/src/exports.ts

```diff
@@ -1 +1,4 @@
-export { useAutoReloadOverlayBlocker } from './components/AutoReloadOverlayBlocker';
+export {
+  useAutoReloadOverlayBlocker,
```

Comment:
This is the change i've made in the SEO plugin to match this

[strapi/strapi-plugin-seo@`df7f255` (#76)](https://github.com/strapi/strapi-plugin-seo/pull/76/commits/df7f255d6162c72ec6153d91eb1d0bf9a5665c86)

Reply:
Thanks for sharing. Not a huge fan of exposing all these details but we don't have a lot of extra time. Do you mind prefixing them with unstable_ to avoid having a wide spread usage until we think this through ?

Reply:
Yep can do that

Im not sure what the best idea is but could we move where this in the monorepo https://github.com/strapi/strapi/blob/ef19c1f78d27a4ed68775c83db061c2d4af07248/packages/core/content-type-builder/admin/src/pages/App/index.tsx#L39-L40

To wrap a higher level of the FE that plugins could have access to?

Reply:
I was thinking the same actually to have a Reload api with a more generic useReload hook that is in the admin for example. Maybe do that after the stable and we just expose as is with `private_` to be even more explicit that it should not be used and then we do the refacto ?